src/append_only_zks.rs

```diff
@@ -120,6 +120,37 @@ impl Azks {
             .await
     }
 
+    async fn preload_nodes_for_insertion<S: V2Storage + Sync + Send, H: Hasher>(
+        &self,
+        storage: &S,
+        _insertion_set: Vec<(NodeLabel, H::Digest)>,
```

Comment:
Unused? For the next-optimization, we'll be doing a runtime breadth-first-search for prefetching (greedily), so I don't think we'd use the optimization set for some kind of preload.

Reply:
Oh, we meant for this function to do all the necessary preloads, this is a starter draft version :)

Reply:
In that case we should also borrow it and not clone it :) (since it too is the size of the publish op (100K+))